torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -483,18 +510,6 @@ at::Tensor AtenXlaType::add(const at::Tensor& self, const at::Scalar& other,
                     });
 }
 
-at::Tensor& AtenXlaType::add_(at::Tensor& self, const at::Tensor& other,
-                              const at::Scalar& alpha) {
-  XLA_FN_COUNTER("xla::");
-  at::native::alpha_check(at::result_type(self, other), alpha);
-  CheckBinaryOpTypePromotion(self, self, other);
-  XLATensor self_tensor = bridge::GetXlaTensor(self);
-  XLATensor::add_(self_tensor,
```

Comment:
actually, I should have removed the code for `XLATensor::add_` in this PR too. I'm planning a followup PR where I remove most other inplace ops, so I'll remove it there too.